Spigot-Server-Patches/0246-Port-Sponge-Async-lighting-patch-to-Paper.patch

```diff
@@ -0,0 +1,1221 @@
+From 735f18db73f89390c9989a339884a42dff9c8896 Mon Sep 17 00:00:00 2001
+From: Mark Vainomaa <mikroskeem@mikroskeem.eu>
+Date: Tue, 17 Oct 2017 00:32:05 +0300
+Subject: [PATCH] Port Sponge Async lighting patch to Paper
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/PaperConfig.java b/src/main/java/com/destroystokyo/paper/PaperConfig.java
+index f5cb9799b..1ce0e3805 100644
+--- a/src/main/java/com/destroystokyo/paper/PaperConfig.java
++++ b/src/main/java/com/destroystokyo/paper/PaperConfig.java
+@@ -276,4 +276,9 @@ public class PaperConfig {
+     private static void authenticationServersDownKickMessage() {
+         authenticationServersDownKickMessage = Strings.emptyToNull(getString("messages.kick.authentication-servers-down", authenticationServersDownKickMessage));
+     }
++
++    public static int asyncLightingThreads = 2;
++    private static void asyncLightingThreads() {
++        asyncLightingThreads = getInt("settings.async-lighting-threads", 2);
```

Comment:
I'm actually in favor of not making this configurable at all.

It's going to lead to people thinking more threads = better.